automatic/totalcommander/TotalCommander.nuspec

```diff
@@ -0,0 +1,73 @@
+* Quick View panel with bitmap display
+* ZIP, 7ZIP, ARJ, LZH, RAR, UC2, TAR, GZ, CAB, ACE archive handling + plugins
+* Built-in FTP client with FXP (server to server) and HTTP proxy support
+* Parallel port link, multi-rename tool
+* Tabbed interface, regular expressions, history+favorites buttons
+* Thumbnails view, custom columns, enhanced search
+* Compare editor, cursor in lister, separate trees, logging, enhanced overwrite dialog etc.
+* Unicode names almost everywhere, long names (&gt;259 characters), password manager for ftp and plugins, synchronize empty dirs, 64 bit context menu, quick file filter (Ctrl+S)
+* USB port connection via special [direct transfer cable](http://ghisler.com/cables/index.htm), partial branch view (Ctrl+Shift+B), and many improvements to ftp, synchronizing and other functions
+* [And many more!](http://ghisler.com/featurel.htm)
+
+## Package parameters
+
+The following package parameters can be set:
+
+* `/LocalUser` - Install only for local user. By default it will be installed for all users.
+* `/DesktopIcon` - Add an icon for Total Commander to the Desktop. By default no icon is added.
+* `/InstallPath` - Use custom install path. By default Total Commander is installed to the `%ProgramFiles%\totalcmd` directory.
+
+These parameters can be passed to the installer with the use of `--params`.
+For example: `--params '/LocalUser /DesktopIcon'`.
+
+## Notes
+
+Total Commander is a Shareware program. This means that you can test it for a period of 30 days. After testing the program, you must either [order the full version](http://www.ghisler.com/order.htm), or delete the program from your harddisk.
+
+[FAQ](http://www.ghisler.com/faq.htm)
+[Plugins](http://www.ghisler.ch/wiki/index.php/Developer%27s_corner)
+[Plugin downloads](http://totalcmd.net/)
+[Plugin development](http://totalcmd.net/directory/developer.html)
+    </description>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/totalcommander</packageSourceUrl>
+    <projectUrl>http://www.ghisler.com</projectUrl>
+    <bugTrackerUrl>http://www.ghisler.ch/wiki/index.php/Known_bugs_and_workarounds</bugTrackerUrl>
+    <docsUrl>http://www.ghisler.ch/wiki/index.php</docsUrl>
+    <mailingListUrl>http://www.ghisler.ch/board/index.php</mailingListUrl>
+    <tags>file manager shareware nagscreen admin</tags>
+    <copyright>© 1995 Christian Ghisler</copyright>
+    <licenseUrl>http://www.ghisler.com/order.htm</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/4d9fd7c774e1ae7edb688d04ac9b55b87477e318/icons/totalcommander.png</iconUrl>
+    <releaseNotes>http://ghisler.com/whatsnew.htm</releaseNotes>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.7" /> 
+      <dependency id="7zip.portable" />
+      <dependency id="autohotkey.portable" />
```

Comment:
as per [1.1.11 Dependency versions](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#1111-dependency-versions) please specify a minimum version for both 7zip.portable and autohotkey.portable